examples/lighting-app/linux/ui.cpp

```diff
@@ -213,12 +255,39 @@ void DeviceState::ChipLoopUpdate()
 
     // TODO:
     //    - consider error checking
-    //    - add more attributes to the display (color? brightness?)
     {
         uint8_t value;
-        emberAfReadServerAttribute(kLightEndpointId, chip::app::Clusters::OnOff::Id,
-                                   chip::app::Clusters::OnOff::Attributes::OnOff::Id, &value, sizeof(value));
-        mOnOff = (value != 0);
+        emberAfReadAttribute(kLightEndpointId, chip::app::Clusters::OnOff::Id, chip::app::Clusters::OnOff::Attributes::OnOff::Id,
+                             &value, sizeof(value));
+        mLightIsOn = (value != 0);
+
+        // Level Control
+        emberAfReadAttribute(kLightEndpointId, chip::app::Clusters::LevelControl::Id,
```

Comment:
Please use the type-safe attribute accessors, not `emberAfReadAttribute`.

Reply:
Updated - looks a lot cleaner!